src/app/util/ember-print.h

```diff
@@ -27,18 +27,24 @@
  * @brief Prints a log message
  * @param category - Currently ignored as zcl categories do not map to chip categories. Defaults to kLogCategory_Progress
  * @param format - Format string to print
- * */
+ */
+#if CHIP_PROGRESS_LOGGING
 void emberAfPrint(int category, const char * format, ...) ENFORCE_FORMAT(2, 3);
+#else
+#define emberAfPrint(...) ((void) 0)
+#endif
 
 /**
  * @brief Prints a log followed by new line line
  * @param category - Currently ignored as zcl categories do not map to chip categories. Defaults to kLogCategory_Progress
  * @param format - Format string to print
- * */
+ */
 #if CHIP_PW_TOKENIZER_LOGGING
 #define emberAfPrintln(MOD, MSG, ...) ChipLogProgress(Zcl, MSG, __VA_ARGS__);
```

Comment:
Shouldn't this be nested within `#if CHIP_PROGRESS_LOGGING` to match the changes you made in `ember-print.cpp`?

Reply:
Based on the conditions in the header, the definition in `ember-print.cpp` is only needed if `CHIP_PROGRESS_LOGGING` && !CHIP_PW_TOKENIZER_LOGGING` so everything looks good to me. Or you think it may not work with some configuration?

Reply:
Resolving it for now, but let me know if I missed something and I'll create a followup PR.